src/collections/programs/hacktoberfest-2020/index.mdx

```diff
@@ -96,61 +96,53 @@ import hacktoberfest2020 from "./hacktoberfest2021.png";
           href="https://youtu.be/wK7Q-zbJ3gQ?t=3" target="_blank">recording</a></td>
-      <td><a href="/community/members/nupur-thakur">Nupur Thakur</a></td>
+      <td><a href="https://layer5.io/community/members/nupur-thakur">Nupur Thakur</a></td>
     </tr>
     <tr>
       <td>Oct. 15th, 2020</td>
       <td><i>A tutorial on contributing to Layer5 and working with Git </i></td>
       <td><a
           href="https://docs.google.com/presentation/d/1UOlwFtZ-VJhW4RgQoSI_QOGUQ7OBG8NZDWrFUor7eyY/edit?usp=sharing" target="_blank">slides</a>, <a
           href="https://youtu.be/O9UZO5g9BvI" target="_blank">recording</a></td>
-      <td><a href="/community/members/ruth-ikegah">Ruth Ikegah</a></td>
+      <td><a href="https://layer5.io/community/members/ruth-ikegah">Ruth Ikegah</a></td>
     </tr>
     <tr>
       <td>Oct. 22nd, 2020</td>
       <td><i>An introduction to all Layer5 repositories</i></td>
       <td><a
           href="https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit" target="_blank">slides</a>, <a
           href="https://www.youtube.com/watch?v=Yu03slXrdS0&list=PL3A-A6hPO2INVEknQ4R-VwDMUT9yM_FSm&index=8" target="_blank">recording</a></td>
-      <td><a href="/community/members/lee-calcote">Lee Calcote</a></td>
+      <td><a href="https://layer5.io/community/members/lee-calcote">Lee Calcote</a></td>
     </tr>
     <tr>
       <td>Oct. 29th, 2020</td>
       <td><i>A tutorial on Gatsby</i></td>
       <td><a
           href="https://www.gatsbyjs.com/docs/quick-start/" target="_blank">slides</a>, <a
           href="https://www.youtube.com/watch?v=ug6yaYC-Kkw&list=PL3A-A6hPO2INVEknQ4R-VwDMUT9yM_FSm&index=9" target="_blank">recording</a></td>
-      <td><a href="/community/members/jash-patel">Jash Patel</a>, <a href="/community/members/tanuj-agarwal">Tanuj Agarwal</a></td>
+      <td><a href="https://layer5.io/community/members/jash-patel">Jash Patel</a>, <a href="https://layer5.io/community/members/tanuj-agarwal">Tanuj Agarwal</a></td>
     </tr>
      <tr>
       <td>Nov. 4th, 2020</td>
       <td><i>Functional Testing with Cypress in Meshery UI</i></td>
       <td><a
           href="https://layer5.io/blog/meshery/functional-testing-with-cypress-in-meshery-ui" target="_blank">slides</a>, <a
           href="https://youtu.be/pIFSI7xtwFs" target="_blank">recording</a></td>
-      <td><a href="/community/members/rodolfo-martinez-vega">Rodolfo Martinez Vega</a></td>
-    </tr>
-     <tr>
-      <td>Jan. 28th, 2021</td>
-      <td><i>Beginner's guide to contributing to Meshery and mesheryctl</i></td>
-      <td><a
-          href="https://docs.google.com/presentation/d/1oUzWQpFeFbpIs_sejtOPbF9J4nigj9ziEAzoWxmm6ig/edit" target="_blank">slides</a>, <a
-          href="https://youtu.be/hh_kFLZx3G4" target="_blank">recording</a></td>
-      <td><a href="/community/members/navendu-pottekkat">Navendu Pottekkat</a></td>
```

Comment:
Nope, you shouldn't be deleting the section. Instead, just remove the hyperlink for Navendu's name. That's all

Reply:
ok I'll fix this